data/strings/types_strings.txt

```diff
@@ -30628,6 +30717,278 @@
+    ca = Atracció històrica
+    cs = Historická atrakce
+    da = Historisk attraktion
+    de = Historische Attraktion
+    el = Ιστορικό αξιοθέατο
+    es = Atracción histórica
+    et = Ajalooline vaatamisväärsus
+    eu = Erakargarri historikoa
+    fa = جاذبه تاریخی
+    fi = Historiallinen nähtävyys
+    fr = Attraction historique
+    he = אטרקציה היסטורית
+    hi = ऐतिहासिक आकर्षण
+    hu = Történelmi látványosság
+    id = Atraksi bersejarah
+    it = Attrazione storica
+    ja = 歴史的アトラクション
+    ko = 역사적인 명소
+    lt = Istorinė atrakcija
+    lv = Vēsturiska atrakcija
+    mr = ऐतिहासिक आकर्षण
+    nb = Historisk attraksjon
+    nl = Historische attractie
+    pl = Atrakcja historyczna
+    pt = Atração histórica
+    ro = Atracție istorică
+    ru = Исторический аттракцион
+    sk = Historická atrakcia
+    sv = Historisk sevärdhet
+    sw = Kivutio cha kihistoria
+    th = แหล่งท่องเที่ยวทางประวัติศาสตร์
+    tr = Tarihi cazibe merkezi
+    uk = Історична пам'ятка
+    vi = Điểm tham quan lịch sử
+    zh-Hans = 历史景点
+    zh-Hant = 歷史景點
+
+  [type.attraction.maze]
+    en = Maze
+    af = Doolhof
+    ar = المتاهة
+    az = Labirent
+    be = Лабірынт
+    bg = Лабиринт
+    ca = Laberint
+    cs = Bludiště
+    da = Labyrint
+    de = Labyrinth
+    el = Λαβύρινθος
+    es = Laberinto
+    et = Labürint
+    eu = Labirintoa
+    fa = ماز
+    fi = Sokkelo
+    fr = Labyrinthe
+    he = מָבוֹך
+    hi = भूलभुलैया
+    hu = Labirintus
+    id = Labirin
+    it = Labirinto
+    ja = 迷路
+    ko = 미로
+    lt = Labirintas
+    lv = Labirints
+    mr = चक्रव्यूह
+    nb = Labyrint
+    nl = Doolhof
+    pl = Labirynt
+    pt = Labirinto
+    ro = Labirint
+    ru = Лабиринт
+    sk = Bludisko
+    sv = Labyrint
+    sw = Maze
+    th = เขาวงกต
+    tr = Labirent
+    uk = Лабіринт
+    vi = mê cung
+    zh-Hans = 迷宫
+    zh-Hant = 迷宮
+
+  [type.attraction.roller_coaster]
+    en = Roller Coaster
+    af = Roller Coaster
+    ar = قطار الملاهي الدوارة
+    az = Roller Coaster
+    be = Амерыканскія горкі
+    bg = Въртележка
+    ca = Muntanya Russa
+    cs = Horská dráha
+    da = Rutsjebane
+    de = Achterbahn
+    el = Τρενάκι του λούνα παρκ
+    es = Montaña rusa
+    et = Vuoristorata
+    eu = Errusiar mendia
+    fa = ترن هوایی
+    fi = Vuoristorata
+    fr = Les montagnes russes
+    he = רכבת הרים
+    hi = रोलर कॉस्टर
+    hu = Hullámvasút
+    id = Roller Coaster
+    it = Montagne russe
+    ja = ジェットコースター
+    ko = 롤러 코스터
+    lt = Amerikietiški kalneliai
+    lv = Amerikāņu kalniņi
+    mr = रोलर कोस्टर
+    nb = Berg- og dalbane
+    nl = Achtbaan
+    pl = Kolejka górska
+    pt = Montanha-russa
+    ro = Rollercoaster
+    ru = Американские горки
+    sk = Horská dráha
+    sv = Berg- och dalbana
+    sw = Roller Coaster
+    th = รถไฟเหาะ
+    tr = Lunapark hız treni
+    uk = Американські гірки
+    vi = Tàu lượn siêu tốc
+    zh-Hans = 过山车
+    zh-Hant = 雲霄飛車
+
+  [type.attraction.train]
+    en = Train attraction
```

Comment:
```suggestion
  [type.attraction.train]
    comment = A trackless vehicle styled as train used to visit e.g. a park and see other attractions or views. 
    en = Trackless Train
```
https://en.wikipedia.org/wiki/Trackless_train

Reply:
I took a quick look at the worldwide _attraction=trains_ with [Overpass](https://overpass-turbo.eu/?w=%22attraction%22%3D%22train%22+global&R) and the usage is quite inconsistent. Sometimes it's used on a node and sometimes on a line, some are probably rather _roller_coasters_ and there is only 815 of them.
I would rather leave this tag out.